aries-site/src/examples/components/rangeinput/RangeInputDescriptionExample.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { Box, FormField, RangeInput, Text } from 'grommet';
+
+export const RangeInputDescriptionExample = () => {
+  const [value, setValue] = useState(10);
+
+  return (
+    <FormField label="Label" help="RangeInput Description">
+      <Box
+        pad={{ horizontal: '11px', vertical: '5px' }}
+        direction="row"
+        gap="medium"
+        width="large"
+      >
+        <Text weight={600}>0</Text>
```

Comment:
alignment is off, should be vertically aligned centered:
![Screen Shot 2021-09-22 at 9 14 23 AM](https://user-images.githubusercontent.com/1756948/134371723-8942162d-6957-4842-a837-5c3244adbf39.png)

Reply:
I believe the Figma file has the font-weight as "medium" which commonly maps to "500" [reference](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping)

```suggestion
        <Text weight={500}>0</Text>
```